src/assets/scripts/client/navigationLibrary/ProcedureModel.js

```diff
@@ -294,6 +294,38 @@ export default class ProcedureModel {
 
         return exitNames[randomIndex];
     }
+	
+	
+	  /**
+     * Return the name of the first entry point
+     *
+     * @for ProcedureModel
+     * @method getFirstEntryPoint
+     * @return {string}
+     */
+    getFirstEntryPoint() {
```

Comment:
see previous comment about editorconfig/eslint.  I'll hold on a review of this section until then